modules/setting/indexer.go

```diff
@@ -70,15 +70,6 @@ func loadIndexerFrom(rootCfg ConfigProvider) {
 
 	Indexer.IssueIndexerName = sec.Key("ISSUE_INDEXER_NAME").MustString(Indexer.IssueIndexerName)
 
-	// The following settings are deprecated and can be overridden by settings in [queue] or [queue.issue_indexer]
```

Comment:
I don't think we need to remove these lines. It's just warning if users' configuration hasn't been migrated.

Reply:
See the description.

![image](https://user-images.githubusercontent.com/2114189/236656889-1dd388d9-f7e6-4beb-b4bd-e27d7f6dc814.png)